Assets/Scripts/InventoryScripts/JournalInventoryScript.cs

```diff
@@ -1,147 +1,28 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using Dialogue_scripts;
 using UnityEngine;
 using UnityEngine.UI;
 
-public class JournalInventoryScript : MonoBehaviour {
-
-    public Button[] journalItemPlace;                //creates an array of Buttons
-    public Button[] pageItemPlace;
-    public GameObject[] GoodPages;
-    public GameObject[] EvilPages;
-    public GameObject[] DeceitfulPages;
-    public GameObject[] UnfortunatePages;
-    public ManagerScript manager;                    //reference to ManagerScript
+public class JournalInventoryScript : MonoBehaviour
+{
+    [SerializeField] private Button[] pageItemPlace;
+    [SerializeField] public GameObject[] GoodPages;
+    [SerializeField] public GameObject[] EvilPages;
+    [SerializeField] public GameObject[] DeceitfulPages;
+    [SerializeField] public GameObject[] UnfortunatePages;
```

Comment:
public serialized fields. Please pick one or the other.

Reply:
Some areas I have used public serialized fields as they are required to be access by other types. Serialized field personally preferable to be used, but public is necessary here. Advice?

Reply:
Properties. Use the fields as backing fields and keep them private.

Reply:
Example:
```
[SerializeField} private GameObject[] _goodPages;

public GameObeject[] GoodPages => _goodPages;